console/backend/src/main/java/org/frankframework/console/filters/CorsFilter.java

```diff
@@ -40,12 +43,12 @@ public class CorsFilter implements Filter {
 	private final Logger log = LogManager.getLogger(this);
 
 	@Value("${iaf-api.cors.allowOrigin:*}")
-	private String allowedCorsOrigins; //Defaults to ALL allowed
+	private String allowedCorsOrigins; // Defaults to ALL allowed
 
 	@Value("${iaf-api.cors.exposeHeaders:Allow, ETag, Content-Disposition}")
 	private String exposedCorsHeaders;
 
-	//TODO: Maybe filter out the methods that are not present on the resource? Till then allow all methods
+	// TODO: Maybe filter out the methods that are not present on the resource? Till then allow all methods
```

Comment:
TODO's get overlooked a lot of the times. Should we make an issue for this?

Reply:
Niet hiervoor, staat standaard toch uit.